twake/frontend/src/app/features/global/services/search-list-manager-service.ts

```diff
@@ -35,8 +35,9 @@ class SearchListManager extends Observable {
 
```

Comment:
So what is this file used for now ?

Reply:
`WorkspaceChannelList.tsx` and `SearchListContainer.tsx` for cmdOrCtrl+K.

We basically use it to transform data into generic channels